subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/InstantExecutionBuildOperations.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.instantexecution
+
+import org.gradle.internal.operations.BuildOperationContext
+import org.gradle.internal.operations.BuildOperationDescriptor
+import org.gradle.internal.operations.BuildOperationExecutor
+import org.gradle.internal.operations.BuildOperationType
+import org.gradle.internal.operations.RunnableBuildOperation
+
+
+class InstantExecutionStateLoadBuildOperationType private constructor()
```

Comment:
who's going to consume these? we should only add a type + details + results if there is a concrete consumer

Reply:
No consumer. Thanks, I simplified the operations